nestkernel/connection_manager.cpp

```diff
@@ -464,6 +464,18 @@ nest::ConnectionManager::connect( const index sgid,
   {
     connect_from_device_( *source, *target, target_thread, syn_id, params, delay, weight );
   }
+  // globally receiving devices, e.g. volume transmitter
+  else if ( not target->has_proxies() and not target->local_receiver() )
+  {
+    // we do not allow to connect a device to a global receiver at the moment
+    if ( not source->has_proxies() )
+    {
+      throw IllegalConnection( "We do not allow to connect a device to a global receiver at the moment" );
+      return;
```

Comment:
Maybe remove the `return` as I don't think it makes sense to return after an exception has been thrown?